internal/server/network/driver_ovn.go

```diff
@@ -990,7 +990,16 @@ func (n *ovn) allocateUplinkPortIPs(uplinkNet Network, routerMAC net.HardwareAdd
 					return fmt.Errorf(`Missing required "ipv4.ovn.ranges" config key on uplink network`)
 				}
 
-				ipRanges, err := parseIPRanges(uplinkNetConf["ipv4.ovn.ranges"], uplinkNet.DHCPv4Subnet())
+				dhcpSubnet := uplinkNet.DHCPv4Subnet()
+				allowedNets := []*net.IPNet{}
```

Comment:
Optional: Would it be possible to dispense with the array and just use a `*net.IPNet` instead? Could default to `uplinkIPv4Net` and change the pointer if dhcpSubnet isn't nil.